FormElement.inc

```diff
@@ -59,6 +59,14 @@ class FormElement implements ArrayAccess {
   protected $index;
 
   /**
+   * Holds references to the original form default value for the form
+   * element.
+   *
+   * @var ReadOnlyProtected
```

Comment:
@willtp87, thanks for testing, makes sense since the form is initialised after the alter. @matthewperry if it's OK with you i will wait after the commiters call, in case some one else has any suggestions. If not i'm :+1: for merging and can do it if you are all OK with this.

Thanks to both :smile: !